src/app/[locale]/(home)/occasions/_components/image-dialog.tsx

```
@@ -0,0 +1,41 @@
+"use client";
+
+import Image from "next/image";
+import { Button } from "@/components/ui/button";
+import { Dialog, DialogContent, DialogHeader, DialogTitle, DialogTrigger, DialogClose } from "@/components/ui/dialog";
```

ESLint

[error] 'DialogClose' is defined but never used.

(@typescript-eslint/no-unused-vars)